dizoo/mujoco/config/halfcheetah_gcl_config.py

```diff
@@ -46,7 +46,14 @@
             auto_alpha=False,
         ),
         collect=dict(
-            demonstration_info_path='path',
+            # Users should add their own model path here. Model path should lead to a model.
+            # Absolute path is recommended.
+            # In DI-engine, it is ``exp_name/ckpt/ckpt_best.pth.tar``.
+            model_path=
+            '/Users/nieyunpeng/Documents/files_to_be_delete/lunarlander_doc/halfcheetah_sac_seed0/ckpt/ckpt_best.pth.tar',
```

Comment:
use placeholder, rather you personal file path

Reply:
thx, being a little bit sloppy